src/main/java/crazypants/enderio/material/MaterialRecipes.java

```diff
@@ -60,6 +60,18 @@ public static void registerOresInDictionary() {
     OreDictionary.registerOre("sand", new ItemStack(Blocks.sand, 1, OreDictionary.WILDCARD_VALUE));
     OreDictionary.registerOre("ingotIron", Items.iron_ingot);
     OreDictionary.registerOre("ingotGold", Items.gold_ingot);
+	OreDictionary.registerOre("blockIron", Blocks.iron_block);
```

Comment:
This is not where we do standard oredict registrations, and most of these are already done by forge. See [OreDictionary.java](https://github.com/MinecraftForge/MinecraftForge/blob/1.7.10/src/main/java/net/minecraftforge/oredict/OreDictionary.java#L58-L128).

If you have any extras to add do it [here](https://github.com/SleepyTrousers/EnderCore/blob/master/src/main/java/com/enderio/core/common/OreDict.java),